src/pages/index.js

```diff
@@ -1,38 +1,40 @@
 // ***********************************   DECLARING VARIABLES   ************************************************************
 //IMPORTS
 import './index.css';
-import {initialCards, editButton, profileNameClass, profileDescriptionClass, addPlaceButton, cardModalForm, profileModalForm, cardContainerClass} from '../utils/constants.js';
-import Section from '../components/Section.js';
-import {makePopUpVisible, makeAddPlacePopUpVisible, createNewUserInfo, createFormValidator, createPopupWithForm, createPopupWithImage, createNewCard} from '../utils/utils.js';
+import {editButton, profileNameClass, profileDescriptionClass, addPlaceButton, cardModalForm, profileModalForm, avatarModalForm, avatarButton, profileModal, cardModal, avatarModal} from '../utils/constants.js';
+import {makePopUpVisible, makeAddPlacePopUpVisible, createNewUserInfo, createFormValidator, createPopupWithForm, createPopupWithImage, makeAvatarPopUpVisible} from '../utils/utils.js';
+import Api from '../components/Api.js'
 
 
+const api = new Api();
+api.loadUserInfo();
+api.loadCards();
+
 
 //STORING CLASS INSTANCES
 const imagePopup = createPopupWithImage('#card-modal');
 const userInfo = createNewUserInfo(profileNameClass, profileDescriptionClass);
 const formValidatorProfileModal = createFormValidator(profileModalForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```